File: src/main/java/net/ccbluex/liquidbounce/ui/other/TipSoundPlayer.java
package net.ccbluex.liquidbounce.ui.other;

import javax.sound.sampled.*;
import java.io.File;

public class TipSoundPlayer {
    private final File file;

    public TipSoundPlayer(File file){
        this.file=file;
    }

    public void asyncPlay(){
        new Thread(this::playSound).start();
    }

    public void playSound() {
        try {
            AudioInputStream audioInputStream = AudioSystem.getAudioInputStream(file);
            Clip clip = AudioSystem.getClip();
            clip.open(audioInputStream);
            clip.start();
        } catch (Exception ex) {
            System.out.println("Error with playing sound.");
            ex.printStackTrace();
        }
    }
}

File: src/main/java/net/ccbluex/liquidbounce/ui/sound/TipSoundPlayer.kt
package net.ccbluex.liquidbounce.ui.sound

import java.io.File
import javax.sound.sampled.AudioSystem

class TipSoundPlayer(private val file: File) {
    fun asyncPlay() {
        Thread { playSound() }.start()
    }

    fun playSound() {
        try {
            val audioInputStream = AudioSystem.getAudioInputStream(file)
            val clip = AudioSystem.getClip()
            clip.open(audioInputStream)
            clip.start()
        } catch (ex: Exception) {
            println("Error with playing sound.")
            ex.printStackTrace()
        }
    }
}